### ISSUE TITLE
Additional augmentations

### ISSUE THREAD

**Issue by jankrepl** [jankrepl/deepdow#48]
Rather than reinventing the wheel one could just use `torchvision` transforms  https://pytorch.org/docs/stable/torchvision/transforms.html

- `Compose` (already recreated in `deepdow`)
- `RandomAffine` - scaling and translation along the y axis (lookback) could be a brilliant augmentation for `deepdow` tensors
- `RandomHorizontalFlip` - flipping the time flow
- `Normalize` - a must together with some helper function that computes means, stds in the training set. However, it still assumes that the time series is stationary.
- `RandomErasing` - (similar to the current `Dropout` however it is contiguous regions)

Additionally, torchvision might be also helpful in other tasks (see #39)

The clear downside is introducing yet another dependency. Additionally, one might argue that it is better to go all the way and use `imgaug`, `albumentation`,...
